modules/rollup-plugin-workbox-inject/package.json

```diff
@@ -20,7 +21,7 @@
   },
   "license": "Apache-2.0",
   "peerDependencies": {
-    "workbox-build": "^5.0.0"
+    "rollup": "^1.2.0"
```

Comment:
I'm pretty sure that `rollup` should be listed here, not `workbox-build`. (`workbox-build` is an actual runtime dependency.)

1.2.0 seems to be the minimum version required by `@rollup/plugin-replace`.